src/containers/GradebookPage/index.jsx

```diff
@@ -24,8 +26,9 @@ function shouldShowSpinner(state) {
   return true;
 }
 
-const mapStateToProps = state => (
+const mapStateToProps = (state, ownProps) => (
```

Comment:
Yeah, one wants to be careful about depending on ownProps for mapDispatchToProps (since it can cause a bunch of function to be re-bound if you define them in-line), but for the state one there's little-to-no downside